include/config/config_debug_toggles.h

```diff
@@ -0,0 +1,82 @@
+#ifndef CONFIG_DEBUG_TOGGLES_H
+#define CONFIG_DEBUG_TOGGLES_H
+
+/******************
+ * DEBUG FEATURES *
+ ******************/
+
+/**
+ * Enable or disable debug rom's features
+ * See `config_debug_controls.h` for controls
+ */
+
+// -------------------------------------------
+
+/**
+ * General features
+ */
+
+/**** [NO CLIP] ****/
+#define ENABLE_NO_CLIP
+
+/**** [INPUT DISPLAY] ****/
+// #define SHOW_INPUT_DISPLAY
+
+/**** [CUTSCENE FEATURES] ****/
+// #define SHOW_CS_INFOS
+// #define ENABLE_CS_CONTROL
+
+// Allow skip on the title screen (if using ``ENABLE_CS_CONTROL``)
+#define CS_CTRL_SKIP_TITLE_SCREEN false
+
+/**** [TIME INFORMATIONS] ****/
+// #define SHOW_TIME_INFOS
+
+/**** [FRAMERATE OPTIONS] ****/
+// TODO: check for frame advance for drawing, improve how it works
+#define ENABLE_FRAMERATE_OPTIONS
+
+/**** [MAP SELECT] ****/
+// #define ENABLE_MAP_SELECT
```

Comment:
For this, we should probably ifdef the entire map select gamestate, meaning that it should be ifdef'd out in the spec as well, and in the gamestate table.

Reply:
I already did that, I added ``GAMESTATE_NONE`` as a temp workaround since the table has hardcoded indices

Reply:
Yup ended up realizing that later on in my review and forgot to remove this comment. As an aside we ought to completely redo that table. I did so in my current hack I'm working on and made it work in a much more sane way lol